zul/src/org/zkoss/zul/Borderlayout.java

```diff
@@ -116,9 +120,16 @@ public Borderlayout() {
 	}
 
 	private static boolean isDefaultAnimationDisabled() {
-		if (_defAnimation == null)
-			_defAnimation = Boolean
-					.valueOf(Library.getProperty("org.zkoss.zul.borderlayout.animation.disabed", "false"));
+		if (_defAnimation == null) {
+			String oldProperty = "org.zkoss.zul.borderlayout.animation.disabed";

```

Comment:
Suggestion: You can write a mock to test this code.  You can reference B85_ZK_3838Test to autotest rather than Manual testing .